### DISCUSSION TITLE
NCS and OT1.2 build

### DISCUSSION THREAD

**Discussion by guerbasc** [openthread/openthread#8191] [Q&A] [answered]
Hi,
I have the following image of OTBR  "otbr.1.3-20220711.ot_2ce3d3bf0.ot-br_1813352247.img.zip", I'm trying to build it with REFERENCE_PLATFORM=ncs REFERENCE_RELEASE_TYPE=1.2 options using the following commands as describe in README:

REFERENCE_PLATFORM=ncs REFERENCE_RELEASE_TYPE=1.2 IN_CHINA=0 ./script/make-reference-release.bash

The configuration remains unchanged and thread harness auto-discover it as ONTCS13_BR.

I have some questions :
How can i correctly build OTNCS12_BR for this commit 1813352247 ?
How can I change my BR options and where can I see my build result?

**Comment by jwhui** [comment, marked as answer]
> How can i correctly build OTNCS12_BR for this commit 1813352247 ?

The ot-br-posix repository is included as a submodule. You can see the specific commit on GitHub `ot-br-posix @ 1813352`

> How can I change my BR options?

You can change specific build configs by modifying files after you have cloned the repo and update your submodules. Which file depends on the specific config you are looking to change. You can do a `git grep` to see where build configs are specified.

> where can I see my build result?

The [README.md](https://github.com/openthread/ot-reference-release/blob/main/README.md) states: _This will produce a reference release folder in ./build/. The folder will be named after the release type, date and the OpenThread commit id._
